app/src/main/java/com/hedvig/app/feature/keygear/KeyGearValuationActivity.kt

```diff
@@ -1,23 +1,35 @@
 package com.hedvig.app.feature.keygear
 
+import android.content.Context
+import android.content.Intent
 import android.os.Bundle
+import com.hedvig.android.owldroid.type.MonetaryAmountV2Input
 import com.hedvig.app.BaseActivity
 import com.hedvig.app.R
-import com.hedvig.app.feature.keygear.ui.itemdetail.KeyGearValuationViewModelImpl
+import com.hedvig.app.feature.keygear.ui.itemdetail.KeyGearItemDetailViewModel
 import com.hedvig.app.feature.keygear.ui.itemdetail.PurchaseDateYearMonthPicker
 import com.hedvig.app.util.extensions.observe
 import com.hedvig.app.util.extensions.view.setHapticClickListener
+import com.hedvig.app.util.safeLet
 import kotlinx.android.synthetic.main.activity_key_gear_valuation.*
 import org.koin.android.viewmodel.ext.android.viewModel
+import org.threeten.bp.YearMonth
+import timber.log.Timber
 import java.text.DateFormatSymbols
 
 class KeyGearValuationActivity : BaseActivity(R.layout.activity_key_gear_valuation) {
 
-    private val model: KeyGearValuationViewModelImpl by viewModel()
+    private val model: KeyGearItemDetailViewModel by viewModel()
```

Comment:
I think we should not act on this ViewModel - we should move all code specficially related to this screen to the `KeyGearValuationViewModel` 🙂